packages/react-charts/src/components/ChartTheme/themes/bullet-theme.ts

```diff
@@ -108,11 +108,11 @@ export const BulletPrimaryDotMeasureTheme = {
 export const BulletPrimaryNegativeMeasureTheme = {
   group: {
     colorScale: [
-      chart_color_red_100.value,
-      chart_color_red_200.value,
-      chart_color_red_300.value,
-      chart_color_red_400.value,
-      chart_color_red_500.value
+      chart_bullet_negative_measure_ColorScale_100.var,
+      chart_bullet_negative_measure_ColorScale_200.var,
+      chart_bullet_negative_measure_ColorScale_300.var,
+      chart_bullet_negative_measure_ColorScale_400.var,
+      chart_bullet_negative_measure_ColorScale_500.var
```

Comment:
Are we going to use vars for all color pallets? 

Ideally, all color pallets would be implemented similarly. If not, then perhaps this should be reverted?